.changeset-staged/three-ducks-yell.md

```diff
@@ -0,0 +1,17 @@
+---
+"@logto/cli": major
+"@logto/console": major
+"@logto/core": major
+"@logto/schemas": major
+---
+
+drop settings table and add systems table
+
+**BREAKING CHANGES**
+
+- core: removed `GET /settings` and `PATCH /settings` API
+- core: added `GET /configs/admin-console` and `PATCH /configs/admin-console` API
+  - `/configs/*` APIs are config/key-specific now. they may have different logic per key
+- cli: removed `alterationState` and added `adminConsole` in valid `logto db config` keys, since:
```

Comment:
nit: `alterationState` is not removed, right? It's just moved to `systems` table